components/src/core/DrawerLayout/DrawerLayout.tsx

```diff
@@ -1,36 +1,43 @@
 import React, { ReactNode } from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import { makeStyles, Theme, useTheme } from '@material-ui/core/styles';
 import PropTypes from 'prop-types';
 
 export type DrawerLayoutProps = {
     children: React.ReactNode;
     drawer: ReactNode;
 };
 
-const useStyles = makeStyles({
+const useStyles = makeStyles((theme: Theme) => ({
```

Comment:
can we do this in the same fashion we've done elsewhere:
```
const useStyles = makeStyles((theme: Theme) =>
    createStyles({
```

Reply:
Yep I've updated this.